.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,31 +1,21 @@
 ---
 name: Feature request
-about: Suggest an idea for this project
-title: ''
+about: Propose a new feature or an idea for this project
+title: 'Feature request: '
 labels: enhancement
 assignees: ''
 
 ---
 
----
-name: Feature request
-about: Suggest an idea for this project
-
----
-
-## :warning: Is your feature request related to a problem? Please describe
-<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
-
-## :bulb: Describe the solution you'd like
-<!-- A clear and concise description of what you want to happen. -->
 
-## :bar_chart: Describe alternatives you've considered
```

Comment:
I kinda liked this section, not that I think it is mandatory but I think it forces the person suggesting to think about the different potential approaches.

Reply:
That is a good point 👍